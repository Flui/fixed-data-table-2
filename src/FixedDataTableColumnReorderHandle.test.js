import React from 'react';
import FixedDataTableColumnReorderHandle from 'FixedDataTableColumnReorderHandle';
import ShallowRenderer from 'react-test-renderer/shallow';

it('renders correctly', () => {
  const props = {
  }

  const renderer = new ShallowRenderer();
  renderer.render(<FixedDataTableColumnReorderHandle {...props}></FixedDataTableColumnReorderHandle>);
  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});