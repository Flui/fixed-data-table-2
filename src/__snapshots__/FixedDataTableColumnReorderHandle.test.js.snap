// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="fixedDataTableCellLayout_columnReorderContainer"
  onMouseDown={[Function]}
  style={
    Object {
      "height": undefined,
    }
  }
/>
`;
